MPU/include/mpu.h

```diff
@@ -11,158 +11,92 @@
 #include "gpio.h"
 #include "pedals.h"
 #include "orionbms.h"
+#include "Watchdog_t4.h"
 
-class MPU
+enum
 {
-    private:
-        DRIVERIO driverio;
-        GPIO gpio;
-        PEDALS pedals;
-        CASCADIAMC motorController;
-        ORIONBMS bms;
-        
-        bool isShutdown = false;
-        bool ssReady = false;
-
-        Timer ioRead_wait;
-        Timer canTest_wait;
-        Timer spinningCheck_wait;
-        Timer boosting_debounce;
-
-        enum
-        {
-            TRIGGER_FAULT,
-            FAULT_OK
-        };
-        
-        /**
-         * @brief Writes the fault latch whichever condition you pass (see above enumerator)
-         * 
-         * @param status 
-         */
-        void writeFaultLatch(bool status);
-
-        /**
-        * @brief Shuts off the car in the event of an error
-        * @note ****Code stays here until power cycle for safety****
-        * 
-        */
-        void shutOffCar();
-        
+    TRIGGER_FAULT,
+    FAULT_OK
+};
 
-    public:
-        MPU();
+extern DRIVERIO driverio;
```

Comment:
extern somewhat makes sense to me but I will likely need a walkthrough and/or need to play around with this code in VS Code once it's merged to fully understand. I think I am currently just getting a bit lost in the hierarchy

Reply:
Objects can't be directly declared in the header file like how I was doing, I'm basically saying "make a globally available prototype object driverio" like how you make prototype functions, and then I actually declare it in mpu.cpp

Reply:
At least that's my understanding of how it works, it worked for how I did CAN, I found it out from stack overflow so like I'm a bit iffy on it too but I kinda get it